readme.md

```diff
@@ -1,5 +1,113 @@
+
+**Retired** - a project which was once a Member Project but no longer satisfies either the Eligibility or Health requirements, or has submitted a formal request to withdraw from the .NET Foundation.
+
+## Eligibility Criteria
+
+Broken into a variety of functional areas, the eligibility criteria is a checklist which defines if a project is fully aligned with the mission and values of the .NET Foundation.
+
+**Suitability**
+
+- The project operates within moral and ethical standards of the .NET Foundation.
+- The project is aligned with the philosophy and guidelines for collaborative development.
+- The project is built on the .NET platform and/or creates value within the .NET ecosystem.
+- The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
+
+**Code**
+
+- The project produces software for distribution to the public at no charge.
+- The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
+- The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
+- When applicable, project must use reproducible build settings in its toolchain.
+- The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
+- The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
+- The project code signs their artifacts as appropriate.
+
+**Licenses and Copyright**
+
+- The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
+- Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
+- Committers are bound by a Contributor License Agreement.
+- The copyright ownership of everything that the project produces is clearly defined and documented.
+
+**Quality**
+
+- The project is open and honest about the quality of its code.
+- The project has a public issue tracker where the status of any defect can be easily obtained.
+- The project puts a very high priority on producing secure software. 
+- The project has a published Security Policy.
+
+**Community**
+
+- The project has a home page which provides high level information about its status and purpose.
+- The project welcomes contributions from anyone and embraces diversity in all forms.
+- The project strives to be a meritocracy where more rights and responsibilities are granted to contributors who add value to the project.
+- The project has a public communication channel where community members can engage with maintainers.
```

Comment:
I know some projects like prism have a patron based chat system rather than it being open to all. This could cause problems for some projects where the maintainers use a pay gate to avoid being overwhelmed.